app-misc/pdfpc/pdfpc-4.4.0.ebuild

```diff
@@ -0,0 +1,44 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+VALA_MIN_API_VERSION="0.34"
+VALA_MAX_API_VERSION="0.46" # append versions in in sed line if increased
+
+inherit vala cmake
+
+DESCRIPTION="Presenter console with multi-monitor support for PDF files"
+HOMEPAGE="https://pdfpc.github.io"
+SRC_URI="https://github.com/${PN}/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-2+"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="+gstreamer"
+
+RDEPEND="app-text/poppler:=[cairo]
+	dev-libs/glib:2
+	dev-libs/libgee:0.8
+	gnome-base/librsvg
+	gstreamer? ( media-libs/gstreamer:1.0
+		media-libs/gst-plugins-base:1.0 )
+	sys-apps/dbus
+	x11-libs/gtk+:3"
```

Comment:
Unconditional deps before conditional ones please. You could multiline gstreamer to make it more readable.

Reply:
Fixed. I also had overseen some slot deps that I added aswell.